### DISCUSSION TITLE
Edit additional pivot data

### DISCUSSION THREAD

**Discussion by mwagena** [statamic-rad-pack/runway#528] [Feature Requests]
Hi,

Is there a way to edit additional pivot data?
In our case we are working with the following data structure:

Plan
Id
Name

Product
Id
Name

PlanProduct
plan_id
product_id
price

I have managed to get it working with the BelongsToMany setup, only I wasn't able to make the price editable.

Thanks in advance.

Regards.

**Comment by duncanmcclean** [comment]
There isn't currently a way to edit information in the pivot table, since that's not something the current Relationships Fieldtype in Statamic really allows for.

**Reply by mwagena** [reply]
ah, that's too bad. Is implementing such logic on the roadmap?

**Reply by duncanmcclean** [reply]
Not currently, no. Sorry!
